src/app/ReadHandler.cpp

```diff
@@ -652,6 +719,52 @@ CHIP_ERROR ReadHandler::ProcessSubscribeRequest(System::PacketBufferHandle && aP
 
     mExchangeCtx->WillSendMessage();
 
+#if CHIP_CONFIG_PERSIST_SUBSCRIPTIONS
+    auto * subscriptionResumptionStorage = InteractionModelEngine::GetInstance()->GetSubscriptionResumptionStorage();
```

Comment:
can we split this into a separate method? This method is already 82 lines long, this adds 45 more.

Reply:
Done.